pkg/deploy/registry/deploylog/rest.go

```diff
@@ -25,8 +26,12 @@ import (
 	deployutil "github.com/openshift/origin/pkg/deploy/util"
 )
 
-// defaultTimeout is the default time to wait for the logs of a deployment
-const defaultTimeout time.Duration = 20 * time.Second
+const (
+	// defaultTimeout is the default time to wait for the logs of a deployment.
+	defaultTimeout time.Duration = 20 * time.Second
```

Comment:
what error code we return when this timeout is reached? can client retry getting the logs based on the return code?